src/components/Users/Users.js

```diff
@@ -47,8 +49,44 @@ const Users = (props) => {
       </div>
       <div>
         {user.followed
```

Comment:
Consider instead of conditionally using to buttons with different handlers, using one button with two conditional onclick handlers.
<button onclick={ boolean ? handler1 : handler2} />

Reply:
After all, I still need to draw either signed or not signed, and the result will be the same